pkg/reconciler/instances/loader.go

```diff
@@ -17,6 +17,8 @@ import (
 	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/eventing"
 	//import required to register component reconciler 'istio' in reconciler registry
 	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/istio"
+	//import required to register component reconciler 'mock' in reconciler registry
+	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/mock"
```

Comment:
This will also load the mock in KCP as dedicated service. If you need a mock in your unit-test or simliar, please [have a look here](https://github.com/kyma-incubator/reconciler/blob/main/cmd/reconciler/start/service/integration_test.go#L99-L105): this create a component reconciler dynamically during the test execution.

Reply:
I assume that if we moved pr to protected branch we can proceed with this approach